src/components/Table/index.tsx

```diff
@@ -87,12 +93,32 @@ function Table<D, K extends string | number, C extends Column<D, K, any, any>>(
         cellClassName,
         uiMode,
         rowModifier,
+        fixedColumnWidth,
+        resizableColumn,
     } = props;
 
     const themeClassName = useThemeClassName(uiMode, styles.light, styles.dark);
+    const [columnWidths, setColumnWidths] = React.useState<Record<string, number>>({});
+    const handleColumnResize = React.useCallback((width: number, name: string | undefined) => {
+        if (isDefined(name)) {
+            setColumnWidths((prevColumnWidths) => ({
+                ...prevColumnWidths,
+                [name]: width,
+            }));
+        }
+    }, []);
+
+    const width = React.useMemo(() => (
+        sum(columns.map(
+            (column) => columnWidths[column.id] ?? column.columnWidth ?? DEFAULT_COLUMN_WIDTH,
+        ))
+    ), [columns, columnWidths]);
```

Comment:
How about we create a separate mapping with all of the defaults?
Currently there are ways the calculation of width may be inconsistent.